src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -289,6 +288,13 @@ public static JsonObject createFeeFineChargeNoticeContext(Account account, Loan
       .put(FEE_CHARGE, createFeeChargeContext(account, chargeAction));
   }
 
+  public static JsonObject createFeeFineChargeNoticeContextWithoutUser(Account account, Loan loan,
+    FeeFineAction chargeAction) {
+
+    return createLoanNoticeContextWithoutUser(loan)
+      .put(FEE_CHARGE, createFeeChargeContext(account, chargeAction));
+  }
```

Comment:
What if fee/fine is not linked to any loan?
What if it's linked to an item but not to a loan?